source/includes/search/_index.md

```diff
@@ -384,60 +386,74 @@ bby.products('categoryPath.id=abcat0502000&driveCapacityGb!=*',{show:'sku,name,s
-      "salePrice": 14.99
+      "sku": 1752654,
+      "name": "Apple - iPhone 4s 8GB Cell Phone - White (AT&T)"
     },
     {
-      "sku": 24772085,
-      "name": "1 Classic Album &.. - CD",
-      "salePrice": 22.99
+      "sku": 1761045,
+      "name": "Apple - iPhone 4s 8GB Cell Phone - White (Verizon Wireless)"
+    },
+    {
+      "sku": 1729354,
+      "name": "Apple - iPhone 5c 16GB  Cell Phone - Green (AT&T)"
+    },
+    {
+      "sku": 1752291,
+      "name": "Apple - iPhone 5c 16GB Cell Phone - Blue (AT&T)"
     }
+  ]
+}
 ```
 
-When used as part of a string search, the wildcard performs two functions. First, it tokenizes the string, breaking it into words. Second, it operates as a standard wildcard, matching any set of characters in the tokenized string. The following example illustrates both functions. When searching for a string value, you may want to search for variations on a specific word. For example, if you want to find CDs whose titles include the words Classic, Classics or Classical you would use the following query:
+When used as part of a string search, the wildcard performs two functions. First, it tokenizes the string, breaking it into words. Second, it operates as a standard wildcard, matching any set of characters in the tokenized string. The following example illustrates both functions. When searching for a string value, you may want to search for variations on a specific word. 
+
+There are several description attributes by which you can search, including `longDescription`, `shortDescription`, `description` or `name`. There is a single `SKU` attribute to search based on `SKU`.
+
+In this example we are searching the `longDescription` for iPhone&#42;. We have appended iPhone with a wildcard `*` so we can search for iPhones with any suffix. We are also looking for any products that have a SKU with a value of **7619002** - note the **or** `|`. Finally, in our example we have updated the number of results that can be returned per page to **15**. Our search will return page **5** of the total **184** pages. Additional information on how to specify the number of results that should be returned per page and which page to return can be found in our Pagination section.
 
-## Limitations
+## Wildcard - Limitations
 
 + You cannot use a wildcard to begin a string search (e.g. <code>(name=*top)</code>); this type of search is extremely resource intensive and doing so will result in a <code>400</code> error.
 + Wildcard with data is valid for strings only. When used alone, the wildcard can represent any data type. When used with other characters, the wildcard can only represent string data. For example, to find Canon products with customer reviews of 4.x, you cannot use <code>(manufacturer=canon&customerReviewAverage=4.*)</code> as the search string. You would have to use a search string like this: <code>(manufacturer=canon&customerReviewAverage>4&customerReviewAverage<5)</code>.
```

Comment:
Just a minor thing, but this paragraph should probably be either all italics or no italics.